web/src/components/organisms/Dashboard/index.tsx

```diff
@@ -48,11 +50,26 @@ const Dashboard: React.FC<Props> = ({ children, defaultSelectedKeys }) => {
             collapsible
             collapsed={collapsed}
             onCollapse={value => setCollapsed(value)}>
-            <WorkspaceMenu
-              defaultSelectedKeys={defaultSelectedKeys}
-              isPersonalWorkspace={personalWorkspace?.id === currentWorkspace?.id}
-              inlineCollapsed={collapsed}
-              workspaceId={currentWorkspace?.id}></WorkspaceMenu>
+            {menuType === "project" ? (
+              // eslint-disable-next-line react/jsx-no-useless-fragment
+              <>
+                {projectId && (
+                  <ProjectMenu
+                    projectId={projectId}
+                    defaultSelectedKeys={defaultSelectedKeys}
+                    inlineCollapsed={collapsed}
+                    workspaceId={currentWorkspace?.id}
+                  />
+                )}
+              </>
+            ) : (
+              <WorkspaceMenu
+                defaultSelectedKeys={defaultSelectedKeys}
+                isPersonalWorkspace={personalWorkspace?.id === currentWorkspace?.id}
+                inlineCollapsed={collapsed}
+                workspaceId={currentWorkspace?.id}
+              />
+            )}
```

Comment:
```suggestion
            {menuType === "project" ?
                projectId && (
                  <ProjectMenu
                    projectId={projectId}
                    defaultSelectedKeys={defaultSelectedKeys}
                    inlineCollapsed={collapsed}
                    workspaceId={currentWorkspace?.id}
                  />
                )
              </>
            ) : (
              <WorkspaceMenu
                defaultSelectedKeys={defaultSelectedKeys}
                isPersonalWorkspace={personalWorkspace?.id === currentWorkspace?.id}
                inlineCollapsed={collapsed}
                workspaceId={currentWorkspace?.id}
              />
            )}
```

Reply:
@KaWaite , you can't nest conditions like that, adding a fragment is a must here.

Reply:
https://github.com/reearth/reearth-cms/blob/339fdc4cd4270b04c91816c97546e47b97c1a211/web/src/components/organisms/Dashboard/index.tsx#L53-L69